extensions/mssql/src/objectManagement/ui/serverPropertiesDialog.ts

```diff
@@ -45,10 +53,17 @@ export class ServerPropertiesDialog extends ObjectManagementDialogBase<Server, S
 		return ViewServerPropertiesDocUrl;
 	}
 
+	protected override get isDirty(): boolean {
+		return true;
```

Comment:
This should only be true if changes have been made

Reply:
I had to set it to true because changes are not registered when modifying tables

Reply:
That's not good, please file an issue and we can look into it after. Hardcoding it as true definitely isn't what we want going forward.

Reply:
Did you verify the cloned _originalObjectInfo and your objectInfo has the differences after the change? (in objectManagementDialogBase.ts)

Reply:
Actually, this is no longer an issue. I don't remember how I fixed it, but I think it was on the table creation implementation.